packages/dds/sequence/src/sharedIntervalCollection.ts

```diff
@@ -169,11 +170,14 @@ export class SharedIntervalCollection
 		localOpMetadata: unknown,
 	) {
 		if (message.type === MessageType.Operation) {
-			this.intervalCollections.tryProcessMessage(
-				message.contents as IMapOperation,
-				local,
-				message,
-				localOpMetadata,
+			assert(
+				this.intervalCollections.tryProcessMessage(
+					message.contents as IMapOperation,
+					local,
+					message,
+					localOpMetadata,
+				),
+				"SharedIntervalCollection received Invalid op, possibly from a never version",
```

Comment:
```suggestion
				"SharedIntervalCollection received Invalid op, possibly from an unknown code version",
```
I'm guessing you're going for the typescript `never` sense, but this is a bit clearer IMO. Similar change needed on other asserts

Reply:
oh lol--I see you used `newer` elsewhere so I guessed wrong. That's fine instead.